packages/trader/src/AppV2/Components/TradeParameters/TradeTypeTabs/trade-type-tabs.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { observer } from 'mobx-react';
+import clsx from 'clsx';
+import { SegmentedControlSingleChoice } from '@deriv-com/quill-ui';
+import { useTraderStore } from 'Stores/useTraderStores';
+import { TRADE_TYPES } from '@deriv/shared';
+
+type TTradeTypeTabsProps = {
+    is_minimized?: boolean;
+};
+
+const TradeTypeTabs = observer(({ is_minimized }: TTradeTypeTabsProps) => {
+    const { contract_type, is_turbos, is_vanilla, onChange } = useTraderStore();
+    const tab_list = [
+        { label: 'Up', value: TRADE_TYPES.TURBOS.LONG, is_displayed: is_turbos },
```

Comment:
localize?

Reply:
I don't think we translate it on production
<img width="269" alt="Screenshot 2024-07-23 at 8 36 16 AM" src="https://github.com/user-attachments/assets/ce867b20-f2f4-41bd-9b9c-34f1be94b406">
<img width="269" alt="Screenshot 2024-07-23 at 8 36 26 AM" src="https://github.com/user-attachments/assets/6d270910-a49f-4337-90db-fe0e14e7768f">
<img width="269" alt="Screenshot 2024-07-23 at 8 36 32 AM" src="https://github.com/user-attachments/assets/2e07fb53-19d7-4149-8eae-f208a35f715a">
<img width="269" alt="Screenshot 2024-07-23 at 8 36 41 AM" src="https://github.com/user-attachments/assets/fb7b4bfa-6401-4c5c-b2a8-dab0cc26caaa">

Reply:
we don't translate it